features/steps/flaw_list.py

```diff
@@ -151,3 +152,37 @@ def step_impl(context):
     asce_state = home_page.get_specified_cell_value(1, 3)
     assert desc_state == asce_state, "Default search not work."
     context.browser.quit()
+
+
+@given("I assgin an issue to me")
+def step_impl(context):
+    home_page = HomePage(context.browser)
+    detail_page = FlawDetailPage(context.browser)
+    # Get the current username
+    context.user_name = home_page.userBtn.get_text()
+    # Go to a specific flaw detail page
+    go_to_specific_flaw_detail_page(context.browser)
+    # Assign this flaw to the current user
+    detail_page.set_input_field("owner", context.user_name)
```

Comment:
Should we use `Self Assign` button, i remember set email address to this field may failed sometimes.

Reply:
OK